src/index.js

```diff
@@ -11,38 +11,36 @@ const GET_INFO_TIMEOUT = 10000 //ms
 module.exports = (web3, options = {}) => {
   const defaultOptions = {
     ensRegistryAddress: null,
-    providers: {}
+    providers: {},

```

Comment:
For future ease of review, it would be great if the prettier changes were committed separately from the logic changes :).

Reply:
Sorry about that! I use an extension on vscode to automatically format the code 😅